packages/rollup-plugin-index-html/src/process-entry-html.js

```diff
@@ -16,15 +17,27 @@ function processEntryHtml(inputConfig) {
 
   const indexFolder = path.dirname(inputConfig.input);
   const indexHTMLString = fs.readFileSync(inputConfig.input, 'utf-8');
-  const resources = extractResources(indexHTMLString);
-  const modules = resources.jsModules.map(p => path.join(indexFolder, p));
+  const {
+    jsModules,
+    inlineScripts,
+    inlineImportMaps,
+    indexHTML: outputIndexHTML,
+  } = extractResources(indexHTMLString);
+  const input = jsModules.map(p => path.join(indexFolder, p));
+  if (inlineScripts.length) {
+    inlineScripts.forEach(script => {
+      const inlineScriptPath = path.join(indexFolder, `inline.${md5(script)}.js`);
+      fs.writeFileSync(inlineScriptPath, script);
```

Comment:
rollup will already output whatever is in the input, so we don't need to write it ourselves.

Reply:
we need to write here because we're *creating* the input

Reply:
Ouch... I hope there is a cleaner way for thag

Reply:
unless there's some way to pass script content as a string to input, but I think it only takes a list of pathnames

Reply:
could be we could defer all that work and handle it all in generateBundle or writeBundle, which is probably The Right Way™️, but also more tightly couples the thing to rollup